src/disemvowling.js

```diff
@@ -0,0 +1,12 @@
+const readlineSync = require('readline-sync')
+
+const stringString = readlineSync.question('Write a funny sentence\n')
```

Comment:
This function is consise and readable! 

These are minor changes, but I took some time to refactor into a single, exportable function, and renamed a few variables for clarity. Not that you'd need to change yours, just thought it might be useful to see. I'd initially attempted to refactor using map, but it didn't wasn't easy to do.

```js
const userInput = readlineSync.question('Write a funny sentence\n')

const removeVowels = ( inputString ) => {
  const goAwayCharacters = /[aeiou|\s]/
  const stringArray = inputString.split('')
  for( let index = 0; index < stringArray.length; index++ ) {
    if( goAwayCharacters.test(stringArray[index]) ) {
      stringArray.splice( index, 1 )
      index--
    }
  }
  return newString = stringArray.join('')
}

console.log( removeVowels( userInput ))
```